tests/test_integration/test_settings.py.sample

```diff
@@ -0,0 +1,2 @@
+url = 'https://wl-test.example.com/'
```

Comment:
Not a blocking critique... thinking out loud...

This file is producing some side effects:
* we have an awkward note in the readme
* it's being consumed outside of the tests by shell.py

If we had in the root of the project a:
```
configuration.sample.json
```

Then:
* Then `configuration.json` could be consumed by the tests and shell.py
* Yola devs know how to make this file with ease